### PR TITLE
Fix hidden color btn hover effect. Fixes #496

### LINKED ISSUE #496: Settings: "Cancel" link disappears on hover

### PR DESCRIPTION
**Pull request by jfrankl**

### CODE REVIEW

**Comment by maurizi** [comment]
This fixes the original issue, but it makes the text color on the cancel button on the User Profile privacy popup grey on blue instead of whhite on blue.

Is that intended?
screenshot from 2015-03-05 18 52 48

**Comment by jfrankl** [comment]
@maurizi Not intentional! This should be fixed now.

**Comment by maurizi** [comment]
At a glance the code looks fine, but I don't have the time to test this today.  Feel free to reassign this to someone else.

File: src/nyc_trees/apps/survey/templates/survey/reservations_instructions.html [hunk 1]
```diff
@@ -1,38 +1,61 @@
...
+  <h2>Reservations Instructions</h2>
+  <p class="pageheading-description-detail">
+    Before you're able to make reservations you'll need to do the following:
+  </p>
+
+{% endblock aside %}
+
+{% block main %}
+
+  <a {% href="{{ step_info.step.pure_url }}" class="training block item">
```
Comment (jwalgran) [inline src/nyc_trees/apps/survey/templates/survey/reservations_instructions.html]: Looks like this ``<a>`` is a leftover and should be removed.

Comment (jfrankl) [inline src/nyc_trees/apps/survey/templates/survey/reservations_instructions.html]: I don't know what you mean. I think I might need to have a *robot* explain it to me.

Comment (jfrankl) [inline src/nyc_trees/apps/survey/templates/survey/reservations_instructions.html]: (I do know what you mean. But I miss Robot Justin.)

**Comment by jwalgran** [comment]
My groups page got really wonky

screen shot 2015-03-06 at 10 49 33 am

**Comment by jwalgran** [comment]
Event detail page has some dark text buttons

screen shot 2015-03-06 at 10 57 45 am

**Comment by jfrankl** [comment]
Thanks for catching those problems Justin. This should be good now.

**Comment by jwalgran** [comment]
Taking another look

**Comment by jwalgran** [comment]
+1 looking sharp

**Comment by jfrankl** [comment]
So long dear friend bubbles. We'll miss you.

image

**Pull request merged**